src/app/staticanalysis/modules/blockBlockhash.js

```diff
@@ -0,0 +1,29 @@
+var name = 'block.blockhash usage'
+var desc = 'semantics maybe unclear'
+var categories = require('./categories')
+var common = require('./staticAnalysisCommon')
+
+function blockBlockhash () {
+  this.warningNodes = []
+}
+
+blockBlockhash.prototype.visit = function (node) {
+  if (common.isBlockBlockHashAccess(node)) this.warningNodes.push(node)
+}
+
+blockBlockhash.prototype.report = function (compilationResults) {
+  return this.warningNodes.map(function (item, i) {
+    return {
+      warning: `use of "block.blockhash": "block.blockhash" is used to access the last 256 block hashes. A miner computes the block hash be "summing up" the information in the current block mined. By "summing up" the information in a clever way a miner can try to influence the outcome of a transaction in the current block.`,
```

Comment:
Oh sorry, wanted to give an example but could not find a good one, so just drop the `like` :-)